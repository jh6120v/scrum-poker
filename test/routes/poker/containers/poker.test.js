import React from 'react';
import { render } from '@testing-library/react';
import Poker from '../../../../src/routes/poker/containers/poker';

describe('test <Poker />', () => {
    const { container, getByTestId } = render(<Poker />);

    it('test <Poker /> snapshot', () => {
        expect(container.firstChild).toMatchSnapshot();
    });

    it('test <Poker /> and find header', () => {
        const elem = getByTestId('display_header');

        expect(elem.textContent).toBe('Scrum Poker');
    });
});
