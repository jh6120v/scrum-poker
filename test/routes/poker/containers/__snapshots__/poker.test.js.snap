// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Poker /> test <Poker /> snapshot 1`] = `
<div
  class="sc-bdVaJa erSRMf"
>
  <header
    class="sc-bwzfXH kyHYFn"
    data-testid="display_header"
  >
    Scrum Poker
  </header>
</div>
`;
